Core_OverlayMods/Clothes/KoiClothesOverlayGui.cs

```diff
@@ -100,14 +101,29 @@ void ReadTex(string texturePath)
             }
         }
 
-        private static void RefreshInterface(string category = null)
+        internal static void RefreshInterface(string category = null)
         {
             if (!MakerAPI.InsideMaker || _refreshInterfaceRunning || _refreshInterface == null) return;
 
             _refreshInterfaceRunning = true;
             _instance.StartCoroutine(RefreshInterfaceCo(category));
         }
 
+        internal static void RefreshMenuColors(int parts)
```

Comment:
Why is `int parts` passed in but unused in the method?


Instead of having the whole method body in an #if block you can do this instead.
```suggestion
        [Conditional("KK"), Conditional("KKS"), Conditional("EC")]
        internal static void RefreshMenuColors(int parts)
```

Reply:
This doesn't seem to work since the `ChaCustom` class does not exist in HS2 and AI. So it doesn't compile with this method